odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -1814,11 +1814,16 @@ def test_94_image(self):
         self.assertEqual(image_data_uri(record.image_256)[:30], 'data:image/png;base64,iVBORw0K')
 
         # ensure invalid image raises
-        with self.assertRaises(UserError):
+        with self.assertRaises(UserError), self.cr.savepoint():
             record.write({
                 'image': 'invalid image',
             })
 
+        # invalid image does not raise with a new record (use-case: onchange)
+        new_record = record.new(origin=record)
+        new_record.image = '31.54 Kb'
+        self.assertEqual(new_record.image, b'31.54 Kb')
```

Comment:
Isn't this what we wanted to prevent in the first place? If we start to use image tools on this "image" they will crash eventually, instead of having a crash upfront like it was the case currently.

Shouldn't we at least write `False` instead of a string value?